packages/pandoc/build.sh

```diff
@@ -15,11 +15,11 @@ termux_step_make_install() {
 	case "$TERMUX_ARCH" in
 		aarch64)
 			srcurl="https://github.com/jgm/pandoc/releases/download/${TERMUX_PKG_VERSION}/pandoc-${TERMUX_PKG_VERSION}-linux-arm64.tar.gz"
```

Comment:
Are we just downloading the prebuilt binaries?
How's that work, wouldn't that have conflicting hardcoded paths?

Reply:
```sh
$ pandoc -v
pandoc 3.1.13                                                     Features: +server +lua
Scripting engine: Lua 5.4
User data directory: /data/data/com.termux/files/home/.local/share/pandoc                                                           Copyright (C) 2006-2023 John MacFarlane. Web: https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
$ file $PREFIX/bin/pandoc
/data/data/com.termux/files/usr/bin/pandoc: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, BuildID[sha1]=556d94fd1390cbc3ef7b849cce20d13c88ebb5a0, stripped
```

Reply:
So does statically linking it magically fix it or what?
I'm still unclear how the pre-built binary is working on Termux.

Reply:
The executable is statically linked and has no dynamic dependencies or dependencies on external data files.

Reply:
Alright, thanks just wanted to confirm.

Reply:
That's what I was worried about.